### ISSUE TITLE
feature/parseable output

### ISSUE THREAD

**Issue by Shura1oplot** [timothycrosley/deprecated.frosted#4]
I would like to write SublimeText plugin which could use frosted for code checking. Can you add command line option makes output parseable? I need column and keyword as extra values. Keyword will be highlighted in code editor.
Example:

```
test.py:6:1:collections:collections imported but unused
<stdin>:10:4:x:undefined name 'x'
<stdin>:25:8:return:'return' with argument inside generator
```

Or something like this.

**Comment by timothycrosley** [comment]
@Shura1oplot A sublime plugin would be awesome! Will get this in before the 1.1.0 release

**Comment by timothycrosley** [comment]
This has been implemented in the just released version 1.1.0, via a -vb (verbose) command line parameter. 

Thanks!

Timothy
